instances/latest/terminologies/technique/superResolutionMicroscopy.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/superResolutionMicroscopy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": null,
+  "description": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0739445",
+  "knowledgeSpaceLink": null,
+  "name": "super resolution microscopy",
+  "preferredOntologyIdentifier": null,
```

Comment:
add preferred ID from interlex

Reply:
done/deleted instance as already part of other PR